bench/wtperf/wtperf_misc.c

```diff
@@ -134,22 +134,21 @@ backup_read(WTPERF *wtperf, WT_SESSION *session)
     if (ret != 0)
         goto err;
 
+    buf = dmalloc(WT_BACKUP_COPY_SIZE);
     while ((ret = backup_cursor->next(backup_cursor)) == 0) {
         testutil_check(backup_cursor->get_key(backup_cursor, &filename));
 
         rfd = -1;
         /* Open the file handle. */
         len = strlen(wtperf->home) + strlen(filename) + 10;
-        buf = dmalloc(len);
+        testutil_assert(len <= WT_BACKUP_COPY_SIZE);
```

Comment:
Technically, you can eliminate the "len" variable entirely, and just use WT_BACKUP_COPY_SIZE In the snprintf below.

Reply:
I like that. Will fix